aspect/BUILD

```diff
@@ -81,17 +99,7 @@ genrule(
     srcs = ["intellij_info_impl.bzl"],
     outs = ["intellij_info_impl_bundled.bzl"],
     cmd = "cat $(SRCS) >$@ && " +
-          "sed -i -e '/BUNDLED-IGNORE-BEGIN/,/BUNDLED-IGNORE-END/d' $@ && " +
```

Comment:
AFAIK plain `sed -i` [doesn't work on Mac OS](https://stackoverflow.com/a/7573438). I wonder if this aspect has been working correctly in the first place?

Reply:
We no longer release to external users from the google branch, so I think its commits are not relevant.

Reply:
That's a good point, but this code is also present in master.